docs/src/modules/components/MarkdownElement.js

```diff
@@ -257,7 +257,7 @@ const Root = styled('div')(
       '&.MuiCallout-error': {
         color: `var(--muidocs-palette-error-900, ${lightTheme.palette.error[900]})`,
         backgroundColor: `var(--muidocs-palette-error-50, ${lightTheme.palette.error[50]})`,
-        borderColor: `var(--muidocs-palette-error-200, ${lightTheme.palette.error[200]})`,
+        borderColor: `var(--muidocs-palette-error-100, ${lightTheme.palette.error[200]})`,
```

Comment:
```suggestion
        borderColor: `var(--muidocs-palette-error-100, ${lightTheme.palette.error[100]})`,
```
@danilo-leal ideally, the fallback color should be the same `var($main-color, $fallback-color)`.

Reply:
x the others.

Reply:
Sweet, thanks! It's now fixed!